samples/fridge/odFridgeObjects/annotations/15.xml

```diff
@@ -0,0 +1,38 @@
+<annotation>
```

Comment:
any specific reason why we're adding another annotation? 

the image related to this annotation seems to be an empty file

Reply:
Oh, I just saw you commented on the issue:

> I will increase the size of validation set to say 3 and then test again.